lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/adapter/MealsAdapter.kt

```diff
@@ -16,7 +15,8 @@ import com.kinandcarta.lib.find.meal.viewmodel.Meals
 
 class MealsAdapter(
     private var mealsList: List<Meal> = listOf(),
-    private var distanceUnit: DistanceUnit = DistanceUnit.miles
+    private var distanceUnit: DistanceUnit = DistanceUnit.miles,
+    private var clickListener: (String) -> Unit
```

Comment:
Think this can be val since it's not going to be reassigned? Although I might be wrong